runtime/src/iree/hal/drivers/hip/rccl_channel.c

```diff
@@ -592,11 +592,10 @@ iree_status_t iree_hal_hip_nccl_submit_batch(
     iree_hal_collective_batch_entry_t* entry = &batch->entries[i];
     iree_string_view_t collective_str =
         iree_hal_collective_op_format(&entry->op, &string_temp);
-    IREE_HIP_STREAM_TRACE_ZONE_BEGIN_EXTERNAL(
-        tracing_context, tracing_event_list, stream,
-        IREE_HAL_HIP_TRACING_VERBOSITY_FINE, __FILE__, strlen(__FILE__),
-        (uint32_t)__LINE__, __FUNCTION__, strlen(__FUNCTION__),
-        collective_str.data, collective_str.size);
+    IREE_STREAM_TRACE_ZONE_BEGIN_EXTERNAL(
```

Comment:
```suggestion
    IREE_HAL_STREAM_TRACE_ZONE_BEGIN_EXTERNAL(
```
(and `IREE_HAL_GRAPH_TRACE_*` for the others)